lib/Twig/Extension/Core.php

```diff
@@ -1634,4 +1635,31 @@ function twig_get_attribute(Twig_Environment $env, Twig_Source $source, $object,
     return $ret;
 }
 
+function twig_classes(...$args): string
+{
+    $classes = [];
+
+    foreach ($args as $i => $arg) {
+        if (is_string($arg)) {
+            $classes[] = $arg;
+        } elseif (is_array($arg)) {
+            foreach ($arg as $class => $condition) {
+                if (!is_string($class)) {
+                    throw new Twig_Error_Runtime(sprintf('Key #%d of argument #%d should be a string. Got "%s".', $class, $i, gettype($class)));
```

Comment:
For `2.x` branch use `RuntimeError` instead (Same below)

Reply:
Nevermind, this class doesn't use the `2.x` namespace anyway.

Reply:
Thanks Fixed